packages/shared/lib/services/user.service.ts

```diff
@@ -18,6 +18,30 @@ class UserService {
         return result[0];
     }
 
+    async getUserByUuid(uuid: string): Promise<User | null> {
+        const result = await db.knex.select('*').from<User>(`_nango_users`).where({ uuid });
+
+        if (result == null || result.length == 0 || result[0] == null) {
+            return null;
+        }
+
+        return result[0];
```

Comment:
I would use `.first()`
```suggestion
        const result = await db.knex.select('*').from<User>(`_nango_users`).where({ uuid }).first();
        return result || null;
```